bool gripper_controller_switch
---
bool gripper_controller_status